authappliance/menu.py

```diff
@@ -571,6 +707,16 @@ def setup_redundancy(self):
             if code != self.d.DIALOG_OK:
                 return
             else:
+                code = self.d.yesno(
+                    "By default, communication between the MySQL peers is "
+                    "unencrypted. Optionally, the appliance tool can set up "
+                    "an encrypted tinc VPN tunnel between the two peers. "
+                    "Should we set up encrypted master-master replication?",
+                    width=60)
+                if code == self.d.DIALOG_OK:
+                    # TODO: Let user choose the subnet
+                    self.peer.setup_tinc('172.20.1.1', '172.20.1.2', '172.20.1.0/30')
+
```

Comment:
We might need to add a tinc-tunnel-test here. Something may go wrong. And the there is no need to go on with self.peer.setup_redundancy, i think.